README.md

```diff
@@ -28,18 +28,20 @@ Mozilla Sync Storage built with [Rust](https://rust-lang.org).
 
 ## System Requirements
 
+- cmake
+- gcc
+- [golang](https://golang.org/doc/install)
+- libcurl4-openssl-dev
+- libmysqlclient-dev
+- libssl-dev
+- make
+- pkg-config
 - [Rust stable](https://rustup.rs)
-- MySQL 5.7 (or compatible)
-  -\* libmysqlclient (`brew install mysql` on macOS, `apt-get install libmysqlclient-dev` on Ubuntu)
```

Comment:
I think we might want to leave the mac install instructions. We've got some folks who do dev work on macs and have asked how to install mysql.

Reply:
Sure, I'll revert this. Anything else you may want to add?

Reply:
No, I think that's the only bit. Thanks for fixing up this document.